### ISSUE TITLE
decode byte 0xe2 in position: ERROR with a valid biom file.

### ISSUE THREAD

**Issue by MaryoHg** [biocore/biom-format#785]
Dear biom-format team (and users).

I have two biom files which I can't summarize on a local computer. It show the following error:
`$ biom summarize-table -i otu_table_amoa.biom`
>Traceback (most recent call last):
  File "/usr/local/bin/biom", line 11, in <module>
    load_entry_point('biom-format==2.1.6', 'console_scripts', 'biom')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/biom/cli/table_summarizer.py", line 49, in summarize_table
    result = _summarize_table(table, qualitative, observations)
  File "/usr/local/lib/python2.7/dist-packages/biom/cli/table_summarizer.py", line 144, in _summarize_table
    lines.append('%s: ' % k + locale.format('%1.3f', v, grouping=True))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1: ordinal not in range(128)

I employed another computer which has installed the same biom version (v2.1.6) and it was succesfully summarize.
May you help me to fix this (dependendy?) issue?

I appreciate any advice,

Thank you all,

MaryoHg.

**Comment by wasade** [comment]
My guess is that some of the sample IDs use non-ascii characters. Is the other computer using Python 3 by chance (you can determine by running "python --version")?

**Comment by MaryoHg** [comment]
Hi, @wasade.

1. I installed all software(s) in both computers and same version of Python is used;
That's why I can't figure it out what is the issue. I even tried re-installing biom-format package using `pip` (and It didn't work).

`$ python --version`
>Python 2.7.12

2. I used the `$ biom validate-table -i table.biom` command and the output show a valid biom format.

3. I read a post on google showing the same error but I couldn't find a specific solution for this.


I appreciate your help and any further comment.



Maryo.
